serde/src/java/org/apache/hadoop/hive/serde2/avro/AvroSerdeUtils.java

```diff
@@ -88,6 +89,7 @@ public String getPropName(){
   @Deprecated public static final String SCHEMA_NAME = "avro.schema.name";
   @Deprecated public static final String SCHEMA_DOC = "avro.schema.doc";
   @Deprecated public static final String AVRO_SERDE_SCHEMA = AvroTableProperties.AVRO_SERDE_SCHEMA.getPropName();
+  @Deprecated public static final String AVRO_SERDE_MAGIC_BIT = AvroTableProperties.AVRO_SERDE_MAGIC_BYTES.getPropName();
```

Comment:
I don't think this is needed, we can use the enum only.

Reply:
Removed